### ISSUE TITLE
Seed problem

### ISSUE THREAD

**Issue by inluxc** [mercurjs/mercur#5]
I getting the following error:
```
TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator))
    at __values (/home/fulvio/Projects/*****/marketplace/api/node_modules/@medusajs/medusa/dist/commands/seed.js:73:72)
    at /home/fulvio/Projects/*****/marketplace/api/node_modules/@medusajs/medusa/dist/commands/seed.js:455:54
    at step (/home/fulvio/Projects/*****/marketplace/api/node_modules/@medusajs/medusa/dist/commands/seed.js:67:23)
    at Object.next (/home/fulvio/Projects/*****/marketplace/api/node_modules/@medusajs/medusa/dist/commands/seed.js:48:53)
    at fulfilled (/home/fulvio/Projects/*****/marketplace/api/node_modules/@medusajs/medusa/dist/commands/seed.js:39:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:
```

**Comment by JuanEliasJabib02** [comment]
on seed.json file,  create a product:[], this must fix it.

![image](https://github.com/mercurjs/mercur/assets/95072028/11ce5b0a-1ecd-4e34-8609-ebfb53dfef42)

**Comment by cwierzbicki00** [comment]
duplicate of closed #2
